package.json

```diff
@@ -15,11 +15,11 @@
         "@vercel/speed-insights": "^1.0.9",
         "geist": "^1.5.1",
         "katex": "^0.16.22",
-        "next": "^15.5.7",
```

Comment:
Based on this doc, it doesn't to be fixed in the selected revision.

https://x.com/nextjs/status/1999224298591092929?s=46&t=gXoqP5xjZ9aPesNmd6unaA